package.json

```diff
@@ -4,7 +4,7 @@
   "description": "VS Code extension for Apache Daffodil DFDL schema debugging",
   "version": "1.1.0",
   "daffodilVersion": "3.3.0",
-  "omegaEditServerHash": "2f8fec09a67aa2a9d844bcb7c0c0dd54f702186ddaa1eb13e73b1ec6b3cf6d73b687b3e9b35b46da609a14746ff7ec884f7d6abc5ae4ac9ce634235e78c3d56d",
+  "omegaEditServerHash": "ffabce0d095364d1c863466f522dfa0ac2f0574d6394a82c0239a613bb84baf7c7b9fd92fdb0457e1d3c9321249acc21567d7dcfd6ff0528e0e792da1b487407",
```

Comment:
Is there a comment syntax for this json, or some way to explain this hash, like where it comes from, how created, etc. 
Or is that documented elsewhere?

Reply:
So normally this is me downloading the package file manually and running `sha512sum` on it to get this hash value.